src/testMod/resources/data/advancedperipheralstest/computer/tests/modintegrtest.botaniamanapool.lua

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I fixed `isOnEnchantedSoil` on dev/1.19.2. These changes are merged into feat/tests. The function can be tested now

https://github.com/IntelligenceModding/AdvancedPeripherals/commit/1309b60ebc46bf8888ca57c596a389fd90fcef96